crypto/tss/ecdsa/cggmp/signSix/message.go

```diff
@@ -48,14 +46,23 @@ func (m *Message) GetMessageType() types.MessageType {
 	return types.MessageType(m.Type)
 }
 
-func (m *Message) Hash() ([]byte, error) {
-	// NOTE: there's an issue if there's a map field in the message
-	// https://developers.google.com/protocol-buffers/docs/encoding#implications
-	// Deterministic serialization only guarantees the same byte output for a particular binary.
-	bs, err := proto.Marshal(m)
-	if err != nil {
-		return nil, err
+func (m *Message) GetEchoMessage() types.Message {
+	mm := &Message{
+		Type: m.Type,
+		Id:   m.Id,
 	}
-	got := blake2b.Sum256(bs)
-	return got[:], nil
+	switch m.Type {
+	case Type_Round1:
+		mm.Body = &Message_Round1{
+			Round1: &Round1Msg{
+				KCiphertext:     m.GetRound1().GetKCiphertext(),
+				GammaCiphertext: m.GetRound1().GetGammaCiphertext(),
+				Z1:              m.GetRound1().GetZ1(),
+				Z2:              m.GetRound1().GetZ2(),
+				// Not broadcast to all in echo protocol
+				// Psi:             m.GetRound1().GetPsi(),
```

Comment:
The same here ?

Reply:
just a reminder indicating we don't send out Psi in echo message